OpenSim/Sandbox/ExampleHopperDevice/exampleHopperDevice.cpp

```diff
@@ -178,6 +178,7 @@ int main()
     {
         // Build the hopper.
         auto hopper = buildHopper();
+        hopper.finalizeFromProperties(); //required due to known bug
```

Comment:
I will make the requested change, though I think it looks like a bug from the perspective of the model builder. Also note that the only page that comes up in the doxygen when searching for "finalizeFromProperties" is [this one](http://myosin.sourceforge.net/1072/classOpenSim_1_1Component.html#a94531c6dfb2490ef235d05e7911fcc94). Based on this description, it is unlikely someone will know they have to call it after building a model.